<ui version="4.0" >
 <class>ObjectBrowser</class>
 <widget class="QWidget" name="ObjectBrowser" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>419</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QTreeWidget" name="treeWidget" >
     <property name="indentation" >
      <number>20</number>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="headerHidden" >
      <bool>false</bool>
     </property>
     <property name="columnCount" >
      <number>2</number>
     </property>
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>2</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label" >
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>To enable zooming when clicking on tracks/ track states, turn on 'zoom/orient' in 'Interactions'</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="spacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>348</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
